fieldservice_stock/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018 - TODAY, Brian McMaster
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Field Service - Stock",
+    "summary": "Integrate the logistics operations with Field Service",
+    "version": "13.0.1.0.0",
+    "category": "Field Service",
+    "author": "Open Source Integrators, "
+    "Brian McMaster, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/field-service",
+    "depends": [
+        "fieldservice",
+        "stock_request_direction",
+        "stock_request_picking_type",
```

Comment:
missing stock_request_submit in order to do https://github.com/OCA/field-service/pull/671/files?file-filters%5B%5D=.py#diff-f7551e26fc5881da48b15fe64ef2878a3d37ea4fae84aeaa5468416742861325R73